frontend/src/cards/ui/SviAlert.tsx

```diff
@@ -47,9 +49,9 @@ function SviAlert(props: SviAlertProps) {
         <Alert severity="info" className={styles.Alert}>
           This county has a social vulnerability index of <b>{props.svi}</b>;
           which indicates a{" "}
-          <a href={urlMap.cdcSvi} className={color}>
+          <Link to={`${METHODOLOGY_TAB_LINK}#svi`} className={color}>
```

Comment:
This is great man; so glad you got it working. I noticed that just visiting `/methodolgy#svi` directly in the url bar doesn't work... so that's why I wasn't able to use this package to solve the problem a couple months ago I mentioned. But this works great here and I think we can use it lots of other places on the site; especially once our methodology is cleaned up